cryptoinvestor/models/asset.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, id: str, name: str, symbol: str, is_crypto: bool):
         self.symbol = symbol
         self.is_crypto = is_crypto
         self.rates = {}
+        self.coin_status = 10000
```

Comment:
what does this variable represent?

Reply:
Your current wallet status = how much money do you have

Reply:
How much money `asset` has? That doesn't make sense